.generators

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Settings><!--This file was automatically generated by Ruby plugin.
```

Comment:
Is this an IDE file that we checked in on accident?

Reply:
accidentally checked in.